packages/react/src/prefabs/Chat.tsx

```diff
@@ -45,7 +50,10 @@ export function useChat() {
 export function Chat({ messageFormatter, ...props }: ChatProps) {
   const inputRef = React.useRef<HTMLInputElement>(null);
   const ulRef = React.useRef<HTMLUListElement>(null);
-  const { send, chatMessages, isSending } = useChat();
+  const { send, chatMessages, isSending } = useChat({
```

Comment:
This is causing an infinite render loop due to how the object is created. I'd suggest to do something like this instead
```ts
 const chatOptions = React.useMemo(() => {
    return { messageDecoder, messageEncoder };
  }, [messageDecoder, messageEncoder]);
  const { send, chatMessages, isSending } = useChat(chatOptions);
```

Reply:
Thanks, I updated the code with that update

Reply:
Sorry, I should have been clearer on this: the reason why I suggested the `{messageDecoder, messageEncoder}` to be referenced directly in the `useMemo` call, is because the `...props` are further down spread on a `div` element. 
so it makes more sense to do 
```
export function Chat({ messageFormatter, messageDecoder, messageEncoder, ...props }: ChatProps) {
```

Reply:
Ah okay, I fixed that by just adding them in the Chat initial destructuring